cumulusci/robotframework/perftests/short/collection_perf.robot

```diff
@@ -78,14 +81,18 @@ Perftest - Insert 200 Contacts With Addresses
         ...  MailingPostalCode=12345
         ...  Email={{fake.email(domain="salesforce.com")}}
     Salesforce Collection Insert  ${objects}
+    End Perf Timer
 
 Perftest - Insert 200 Prospecting Opportunities
+    Start Perf Timer
```

Comment:
Are you aware that even though this appears before the `[Setup]`, it actually runs after?